rosidl_typesupport_introspection_cpp/resource/msg__type_support.cpp.em

```diff
@@ -49,6 +49,17 @@ namespace @(ns)
 namespace rosidl_typesupport_introspection_cpp
 {
 
+void init_function__@(message.structure.namespaced_type.name)(void * message_memory, bool default_initialize)
```

Comment:
That is correct, I've used a bool as a placeholder because I wasn't exactly aware of the right datastructure/enum to use for the initializers. I'll change that accordingly.
After talking with @clalancette, the default initializers are only yet implemented in C++. I'll change the API for the C datatypes accordingly though to accept a enum as described here: https://github.com/ros2/rosidl/blob/master/rosidl_generator_c/include/rosidl_generator_c/message_initialization.h#L20

Reply:
done in 319d6c7